packages/edit-post/src/components/layout/index.js

```diff
@@ -129,13 +130,23 @@ function Layout( { styles } ) {
 			hasReducedUI: select( editPostStore ).isFeatureActive(
 				'reducedUI'
 			),
+			showBlockBreadcrumbs: select( editPostStore ).isFeatureActive(
+				'showBlockBreadcrumbs'
+			),
 		};
 	}, [] );
+	const shouldShowBlockBreadcrumb =
+		! hasReducedUI &&
+		showBlockBreadcrumbs &&
+		! isMobileViewport &&
+		isRichEditingEnabled &&
+		mode === 'visual';
 	const className = classnames( 'edit-post-layout', 'is-mode-' + mode, {
 		'is-sidebar-opened': sidebarIsOpened,
 		'has-fixed-toolbar': hasFixedToolbar,
 		'has-metaboxes': hasActiveMetaboxes,
 		'show-icon-labels': showIconLabels,
+		'show-block-breadcrumbs': shouldShowBlockBreadcrumb,
```

Comment:
Is this classname necessary?  Both adding this classname and adding the element to the footer of the skeleton depend on the same logic check - `shouldShowBlockBreadcrumb`.  It seems like only one of those things needs to be done and doing both is slightly redundant?

Reply:
See below. I was also thinking about moving this into `InterfaceSkeleton`. If `footer !== null` -> add `has-footer` class to the `InterfaceSkeleton`. What do you think?

Reply:
Thats a good thought!  Since the styles are in the interface skeleton, maybe the class we are applying that they are based off should be as well.  Good point!